src/main/java/ConsoleOut/WaysShow.java

```diff
@@ -0,0 +1,18 @@
+package ConsoleOut;
+
+import java.util.Map;
+
+public class WaysShow {
+    public void showWays(Map<Character, String> ways, int delay, String color){
```

Comment:
Do I want to change color and delay with every call of this method? If not, change it to fields.